packages/aggregate-client/src/types.ts

```diff
@@ -77,3 +71,18 @@ export interface Connectable {
 export type CARLink = Link<unknown, typeof CAR.codec.code>
 
 export type OfferSrc = ToString<URL>
+
+/**
+ * [Piece CID](https://spec.filecoin.io/systems/filecoin_files/piece/) of some
+ * content.
+ */
+export type PieceCID = ReturnType<typeof CommP.toCID>
```

Comment:
Is there not an exported type for this? Or at least can we not define like `CARLink`?

Call this `PieceLink` to be consistent with `CARLink`?

Reply:
I think we want to be consistent with filecoin namings. I stole this from @Gozala 's PR https://github.com/web3-storage/w3up/pull/803 but I actually wondered if `data-segment` lib could actually export the type

Reply:
Yeah it should absolutely export a type, I'll do that